industrial_ci/src/folding/none.sh

```diff
@@ -17,8 +17,10 @@
 
 function  ici_start_fold() {
     shift 3
+    ici_color_output $ANSI_BLUE ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```

Comment:
To make shellcheck happy:
```suggestion
    ici_color_output "$ANSI_BLUE" ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```